### DISCUSSION TITLE
Re-export Prisma Client as a public facing SDK

### DISCUSSION THREAD

**Discussion by kevinslin** [prisma/prisma#13441] [Q&A]
I'm a founder of a developer focused knowledge base and I'm currently researching whether prisma can be embedded into our application.
so far, its checking all the boxes - just one holdout i'm hoping folks here can help me with. 

can a custom prisma client be exported as a npm package so that it can be used as a public facing sdk?
our use case: we offer a plain text based knowledge base for developers and the metadata will be stored in sqlite. 
we have multiple clients (eg. CLI, VSCode, vim) that need to access the metadata - ideally, we can define the prisma model in one place and then export it to all the packages that need it. 

Basically do something like the following and consume this in a separate package

```ts
import { PrismaClient } from '@prisma/client'
export { PrismaClient as DendronClient }
```

Any help would be much appreciated 🙏

**Comment by nurul3101** [comment]
Hey @kevinslin 👋 

I have seen users export the types generated by PrismaClient based on the Prisma Schema in a private npm package and use it in their other apps by importing them. If I am understanding correctly, your use case seems similar to it.

**Reply by joshuaavalon** [reply]
Is there any examples on how to do this? using `output` in `generator client` seems to include the platform specific binary.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
